Docs/source/index.rst

```diff
@@ -3,19 +3,23 @@
 WarpX
 -----
 
-WarpX is an advanced **electromagnetic & electrostatic Particle-In-Cell** code.
+WarpX is an advanced **Particle-In-Cell** code.
 
 It supports many features including:
 
-    - Perfectly-Matched Layers (PML)
-    - Boosted-frame simulations
-    - Mesh refinement
+    - Multiple types of field solvers (incl. electromagnetic, electrostatic)
```

Comment:
In case this is clear enough we can list @roelof-groenewald :
```suggestion
    - Multiple types of field solvers (incl. electromagnetic, electrostatic, kinetic-fluid)
```

Reply:
Actually: the same comment applies to this page:
file:///Users/rlehe/Documents/codes/warpx_directory/warpx/Docs/build/html/theory/kinetic_fluid_hybrid_model.html

Reply:
Technically: "kinetic-fluid" refers to how we treat the particles, not to the type of field equations that we use.

Reply:
I agree the "field solver" is more aptly described as "Ohm's law". But I would say we should overall not paint ourselves into a corner like this by saying "electromagnetic, electrostatic" since much of the versatility of WarpX comes from the zoo of solvers. I would say we should list the various types as, "Maxwell FDTD, Maxwell pseudo-spectral, Poisson, Ohm's law". And then in the future we can add "Darwin".